angular-ui-bootstrap/angular-ui-bootstrap.d.ts

```diff
@@ -5,15 +5,31 @@
 
 /// <reference path="../angularjs/angular.d.ts" />
 
-// Support for AMD require
-declare module 'angular-bootstrap' {
-    let _: string;
-    export = _;
-}
-
+// Support for AMD require and CommonJS
 declare module 'angular-ui-bootstrap' {
-    let _: string;
-    export = _;
+    export default 'ui.bootstrap';
```

Comment:
The module actually uses `module.exports = 'ui.bootstrap';`. So `export =` is correct, not `export default`. This would break people using `import x = require("angular-ui-bootsrap")` currently. (Although, does anyone do that?) You could use `declare namespace _ { export type ... }` to merge a namespace with the export if you want to expose types.